### PR TITLE
[1LP][RFR] Check all stats for Nuage provider

### PR DESCRIPTION
**Pull request by miha-plesko**
With this commit we extend Nuage provider CRUD test to check for all the stats that currently inventoried. Nuage provider details page looks like this:

image

With this PR we now check all the 7 stats.

Depends on: https://github.com/ManageIQ/wrapanapi/pull/326

### CODE REVIEW

File: cfme/networks/provider/nuage.py [hunk 1]
```diff
@@ -153,3 +156,30 @@ def num_network_router(self):
     def num_network_router_ui(self):
         view = navigate_to(self, "Details")
         return int(view.entities.summary("Relationships").get_text_of("Network Routers"))
+
+    @variable(alias="db")
+    def num_cloud_network(self):
```
Comment (RonnyPfannschmidt) [inline cfme/networks/provider/nuage.py]: **required** comment out not yet operational variants to avoid confusion

Comment (miha-plesko) [reply cfme/networks/provider/nuage.py]: Ha, didn't realize I can. Thanks for pointing out, done. I just removed the db variant, no need to have it in as a comment.

**Comment by RonnyPfannschmidt** [comment]
good job, please check the travis failure
**REQUIRED** add comments to consider bringing back the db versions just so we have it documented in the current code, else we might forget it

**Comment by RonnyPfannschmidt** [comment]
oh, i forgot, in case the db versions don't make sense after all, please just tell in which case the comments are no longer needed

**Comment by miha-plesko** [comment]
@RonnyPfannschmidt thanks for comments. As an external contributor I'm unable to see Travis logs at all so you'll have to guide me through if we want to make it green.

Re UI and DB variant: I'm not sure what you want me to comment? From what I understand test will pass if I tell it to use UI variant like here https://github.com/ManageIQ/integration_tests/blob/c3905801a5232aafead7a5a53210cb558a66cde4/cfme/tests/networks/test_network_providers.py#L60

and it will explode if I tell it to use DB variant, which is not implemented. So I'm not sure whether we really need any comment: we will be reminded about the missing implementation loudly, right? 😂 

If you want me to put comment anyway, please let me know.

**Pull request merged**